cmsdb/processes/higgs.py

```diff
@@ -27,36 +27,56 @@
 # Single Higgs
 #
 
+# preliminary Higgs cross sections at 13.6 TeV taken from here:
+# https://cds.cern.ch/record/2886099
+# TODO: h xsecs at 13 TeV
+
 h = Process(
     name="h",
     id=10000,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(0.1),  # TODO
+    },
 )
 
 h_ggf = h.add_process(
     name="h_ggf",
     id=11000,
     label=r"$H_{ggf}$",
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(56.11, {
+            "pdf": 0.021j,
+            "th": (0.046j, 0.067j),
+            "th_gaussian": 0.040,
+        }),  # TODO: only preliminary
+    },
 )
 
 h_ggf_tautau = h_ggf.add_process(
     name="h_ggf_tautau",
     id=11100,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={ecm: h_ggf.get_xsec(ecm) * const.br_h.tt for ecm in (13, 13.6)},
 )
 
 h_vbf = h.add_process(
     name="h_vbf",
     id=12000,
     label=r"$H_{vbf}$",
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(4.075, {
+            "scale": (0.005j, 0.003j),
+            "pdf": 0.021j,
```

Comment:
```suggestion
        13.6: Number(4.078, {  # value for mH=125 GeV
            "scale": (0.005j, 0.003j),
            "pdf": 0.021j,
```


These values are for mH=125.09, however most samples are produced with mH=125. Therefore, I would write the values for the mH=125 GeV cross section (or even better: check which mass was used for the generation, but maybe it is a bit much...). Also, for consistency purposes, staying at 125GeV might be the best option as there is no 125.09 value for 13.6TeV for the di-higgs samples. I would also add a comment to say for which Higgs mass the value was taken (for bookkeeping)